master/buildbot/test/util/decorators.py

```diff
@@ -20,6 +24,31 @@
 from twisted.python import versions
 
 
+_FLAKY_ENV_VAR = 'RUN_FLAKY_TESTS'
+
+
+def todo(message):
+    """
+    decorator to mark a todo test
+    """
+    def wrap(func):
+        """
+        just mark the test
+        """
+        func.todo = message
+        return func
+    return wrap
+
+
+def flaky(bugNumber):
+    def wrap(fn):
+        if not os.environ.get(_FLAKY_ENV_VAR):
+            fn.skip = ("Flaky test (http://trac.buildbot.net/ticket/%d) "
+                       "- set $%s to run anyway" % (bugNumber, _FLAKY_ENV_VAR))
+        return fn
+    return wrap
+
+
 def usesFlushLoggedErrors(test):
```

Comment:
That's a very good point and I think that this PR is very suitable for this removal.